cg/cli/upload/base.py

```diff
@@ -57,29 +58,29 @@ def upload(context: click.Context, case_id: str | None, restart: bool):
     """Upload results from analyses"""
 
     config_object: CGConfig = context.obj
-    upload_api: UploadAPI = MipDNAUploadAPI(config=config_object)  # default upload API
+    upload_api: UploadAPI = MipDNAUploadAPI(config_object)
```

Comment:
Feels like it would make more sense to instantiate with the Parent class and handle MipDNA as the other pipelines. Out of scope though.